opennms-container/horizon/container-fs/confd/templates/newts.properties.tpl

```diff
@@ -1,8 +1,11 @@
 #
 # DON'T EDIT THIS FILE :: GENERATED WITH CONFD
 #
-org.opennms.newts.config.hostname={{getv "/opennms/cassandra/hostnames" "localhost"}}
-org.opennms.newts.config.keyspace={{getv "/opennms/cassandra/keyspace" "newts"}}
-org.opennms.newts.config.port={{getv "/opennms/cassandra/port" "9042"}}
-org.opennms.newts.config.username={{getv "/opennms/cassandra/username" "cassandra"}}
-org.opennms.newts.config.password={{getv "/opennms/cassandra/password" "cassandra"}}
+
+{{$cassandraPath := "/opennms/cassandra/" -}}
+
+org.opennms.newts.config.hostname={{getv (print $cassandraPath "hostname") "hostname"}}
```

Comment:
Is it `hostname` or `hostnames` here? Can you please verify? Just to make sure we keep it as it was and don't break existing configs.

Reply:
Regarding our existing configs in `opennms.properties` it is `hostname`.